lib/kubernetes-deploy/runner.rb

```diff
@@ -19,11 +19,11 @@ def self.with_friendly_errors
       exit 1
     end
 
-    def initialize(namespace:, environment:, current_sha:, template_folder: nil, context:)
+    def initialize(namespace:, environment:, current_sha:, context:, template_dir: nil)
       @namespace = namespace
       @context = context
       @current_sha = current_sha
-      @template_path = File.expand_path('./' + (template_folder || "config/deploy/#{environment}"))
+      @template_dir = template_dir || File.expand_path("./config/deploy/#{environment}")
```

Comment:
It does, but for ENV variables it's always a good practice to use the absolute path (you can check `env` result on your laptop). The idea is behind ENV is that it's global, and when you set a path you don't know where from the script will execute.

> When this is being run in shipit, the current directory will always be app root, which is the reference point devs setting this option have access to. They won't know the absolute path.

Not sure I understand the concern. I couldn't find any references of `K8S_TEMPLATE_FOLDER` in Shipit too.